SalaryService.Domain/Compensation.cs

```diff
@@ -0,0 +1,42 @@
+﻿using NodaTime;
+
+namespace SalaryService.Domain;
+
+public class Compensation
+{
+    public long Id { get; set; }
+
+    //public CompensationType Type { get; private set; }
+
+    public string Type { get; private set; }
+
+    public string? Comment { get; private set; }
+
+    public double Amount { get; private set; }
+
+    public bool IsPaid { get; private set; }
+
+    public long EmployeeId { get; private set; }
+
+    public Employee Employee { get; set; }
+
+    public Instant DateCreateCompensation { get; private set; }
+
+    public Instant DateCompensation { get; private set; }
+
+    public Compensation(string type, string? comment, double amount, Employee employee, string dateCompensation, bool isPaid = false)
+    {
+        Type = type;
+        Comment = comment;
+        Amount = amount;
+        IsPaid = isPaid;
+        Employee = employee;
+        DateCreateCompensation = SystemClock.Instance.GetCurrentInstant();
+        DateCompensation = Instant.FromDateTimeUtc(DateTime.SpecifyKind(DateTime.Parse(dateCompensation), DateTimeKind.Utc));
```

Comment:
You can simplify this line if `dateCompensation` will have a type `Instant`

Reply:
i'll do it later